packages/uui-sort-symbol/lib/uui-sort-symbol.element.ts

```diff
@@ -0,0 +1,101 @@
+import { LitElement, html, css } from 'lit';
+import { property } from 'lit/decorators.js';
+import { ActiveMixin } from '@umbraco-ui/uui-base/lib/mixins';
+
+/**
+ *  @element uui-expand-symbol
+ *  @description A symbol indicating weather related composition is sorting(active) and weather the sorting in ascending or descending.
+ *  Please define aria-sort on the header for the composition.
+ *  @property active - Set this boolean to true for then the related composition is sorted.
+ *  @property descending - Set this boolean to true for displaying descending sort is active.
+ */
+export class UUISortSymbolElement extends ActiveMixin(LitElement) {
+  static styles = [
+    css`
+      :host {
+        position: relative;
+        display: inline-block;
+        width: 0.8em;
+        height: 1em;
+        vertical-align: middle;
+      }
+
+      svg {
+        position: absolute;
+        left: 0;
+        top: 50%;
+        width: 0.8em;
+        fill: currentColor;
+        transform-origin: 50% 50%;
+        transition: transform 120ms ease-in-out, opacity 120ms, margin-top 240ms;
+        opacity: 0;
+        margin-top: -0.5em;
+      }
+
+      #up {
+        transform: rotate(180deg);
+      }
+      #up {
+        margin-top: -0.7em;
+      }
+      #down {
+        margin-top: -0.3em;
+      }
+      :host([active]) #up {
+        margin-top: calc(-0.5em - (0.2em * var(--uui-sort-symbol--hover, 0)));
+      }
+      :host([active]) #down {
+        margin-top: calc(-0.5em + (0.2em * var(--uui-sort-symbol--hover, 0)));
+      }
+
+      :host(:hover) {
+        --uui-sort-symbol--hover: 1;
+      }
+      :host(:not([active])) #up {
+        opacity: calc(0.25 * var(--uui-sort-symbol--hover, 0));
+      }
+      :host(:not([active])) #down {
+        opacity: calc(0.25 * var(--uui-sort-symbol--hover, 0));
+      }
+
+      :host([active]:not([descending])) #down {
+        opacity: 1;
+      }
+      :host([active]:not([descending])) #up {
+        opacity: calc(0.25 * var(--uui-sort-symbol--hover, 0));
+      }
+
+      :host([active][descending]) #up {
+        opacity: 1;
+      }
+      :host([active][descending]) #down {
+        opacity: calc(0.25 * var(--uui-sort-symbol--hover, 0));
+      }
+    `,
+  ];
+
+  /**
+   * Turns the arrow around.
+   * @type {boolean}
+   * @default false
+   */
+  @property({ type: Boolean, reflect: true })
+  public active = false;
```

Comment:
We both use the ActiveMixin and have an active property. Is this needed?

Reply:
Oh yes, thats a mistake. I have removed that now